src/yaml.jl

```diff
@@ -35,9 +35,9 @@ function dftramo_run(filename::AbstractString, software::AbstractString="vasp")
             r.name,
             DFTRAMO_EHT_PARAMS,
             occ_states,
-            dftinfo.geo.basis,
-            dftinfo.kpt,
-            length.(collect.(dftinfo.wave.grange)),
+            Electrum.basis(ramoinput),
```

Comment:
If you use a method from a module that was brought it with `using`, you only need to prefix the package name when overloading the function with new methods or if there's a conflict with an exported name from a different module.

Reply:
Resolved in [001b98c](https://github.com/xamberl/DFT-raMO/pull/39/commits/001b98c698962603e5cf15614e573730df3ca5ba)